make package-linux-arm failing

```
# github.com/getlantern/systray
src/github.com/getlantern/systray/systray_linux.c: In function 'nativeLoop':
src/github.com/getlantern/systray/systray_linux.c:37:2: warning: 'return' with no value, in function returning non-void
  return;
  ^
# github.com/getlantern/systray
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -lappindicator3
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -ldbusmenu-glib
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -lgtk-3
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -lgdk-3
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -lpangocairo-1.0
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -lpango-1.0
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -latk-1.0
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -lcairo-gobject
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -lcairo
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -lgdk_pixbuf-2.0
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -lgio-2.0
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -lgobject-2.0
/usr/lib/gcc/arm-linux-gnueabi/5.1.0/../../../../arm-linux-gnueabi/bin/ld: cannot find -lglib-2.0
collect2: error: ld returned 1 exit status
Makefile:146: recipe for target 'docker-linux-arm' failed
make: *** [docker-linux-arm] Error 2
make: *** [package-linux-arm] Error 2
[lantern (valencia)]$ VERSION=2.0.0 make package-linux-arm
```

--- a/Makefile
+++ b/Makefile
@@ -310,7 +310,7 @@ package-linux-amd64: require-version genassets linux-amd64
 package-linux-arm: require-version genassets linux-arm
 	@echo "Generating distribution package for linux/arm..." && \
 	$(call docker-up) && \
-	docker run -v $$PWD:/lantern -t $(DOCKER_IMAGE_TAG) /bin/bash -c 'cd /lantern && VERSION="'$$VERSION'" make docker-package-linux-arm'
+	docker run -v $$PWD:/lantern -t $(DOCKER_IMAGE_TAG) /bin/bash -c 'cd /lantern && VERSION="'$$VERSION'" HEADLESS="1" make docker-package-linux-arm'
 
 package-linux: require-version package-linux-386 package-linux-amd64 
 

